antarest/study/storage/variantstudy/model/dbmodel.py

```diff
@@ -66,6 +66,8 @@ class CommandBlock(Base):  # type: ignore
     version: int = Column(Integer)
     args: str = Column(String())
     study_version: str = Column(String(36))
+    user_id: int = Column(Integer, ForeignKey("identities.id", ondelete="CASCADE"), nullable=True)
```

Comment:
I think we should put set_null instead of cascade as I think we don't want to remove the command if the user is deleted but rather remove the info on who did it. But it's a case that never happens (we never delete user) so it probably doesn't really matter